packages/client/src/utils/id-token.ts

```diff
@@ -15,25 +15,31 @@ const fullfillBase64 = (input: string) => {
   return input;
 };
 
-const IDTokenSchema = s.type({
+const IdTokenClaimsSchema = s.type({
   iss: s.string(),
   sub: s.string(),
   aud: s.string(),
   exp: s.number(),
   iat: s.number(),
+  auth_time: s.optional(s.number()),
```

Comment:
do we need all these fields declared?

Reply:
I find that some of these claims seems to be unused for now,
so I can also do not add them.

But I still added them here according to the documentations [decodeIdToken](https://www.notion.so/silverhand/decodeIdToken-7b3b739c397a431882a2e41bac8782f1) and [ID Token](https://www.notion.so/silverhand/OIDC-ec922f42788a41a69f5e2550a2f7c3fe#4ffdd579db9b473d8876ce870fcdaede)

@wangsijie @xiaoyijun any ideas? Add them now or when needed?

Reply:
I think we'd better declare them, which will be used in the near future.

Reply:
maybe optional?